CHANGELOG.md

```diff
@@ -60,7 +73,7 @@ These should not affect the functionality of the module.
 
 ## [v4.0.0](https://github.com/voxpupuli/puppet-dhcp/tree/v4.0.0) (2020-02-24)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v3.4.0...v4.0.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/5.1.0...v4.0.0)
 
```

Comment:
Hum, the tags dates seem to be inconsistent.  Maybe moving the old entries to HISTORY.md to generate changelog only for new entries would help?

Reply:
Versions without a `v` seems to relate to `theforeman-dhcp`, while the ones with a `v` relate to `puppet-dhcp`.

When browsing a version without the `v`, GitHub warns:

> [!WARNING]
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Removing these tags may help?

Reply:
Oh yeah, that's weird. I don't think I have permission to remove the foreman tags.

Reply:
Oh, no, looks like I can delete the foreman tags from the web interface. I'll do that.

Reply:
@smortex how does that look now? Looks good to me.